src/spec.json

```diff
@@ -72,7 +73,6 @@
                 {
                     "id": 8,
                     "title": "Use appropriate tools (eg. mapping, financial models)",
-                    "description": "This is specifically asking in a business development context"
```

Comment:
@jmstach I was trying to differentiate between tooling around building the product (writing software, say) vs tools that the Leadership uses to make decisions (maps, models, etc)

Reply:
Understood, I wasn’t so sure that the domain is so specific - compare with other questions and I think it’s looking at the organisational level, so all tools regardless of which function uses them. Note that this isn’t an informed position, I’m just interpreting.